src/charm.py

```diff
@@ -180,9 +191,9 @@ def _render_conf_file(self) -> str:
 
         rendered = template.render(
             db_info=self._get_database_relation_info(),
-            consent_url=join(self.config.get("login_ui_url", ""), "consent"),
-            error_url=join(self.config.get("login_ui_url", ""), "oidc_error"),
-            login_url=join(self.config.get("login_ui_url", ""), "login"),
+            consent_url=self._get_login_ui_endpoint_info("consent_url"),
```

Comment:
This would be called even if the `ui-endpoint-info` relation doesn't exist it seems, meaning that we'd get `consent_url=None`, `error_url=None`, etc. Is this wanted?